experiments/ClimaCore/sea_breeze/land_rhs.jl

```diff
@@ -1,3 +1,14 @@
+# # Land Model
```

Comment:
Since land and ocean are both thermal slabs, should we combine the docs for these?

Reply:
Sounds good. And I guess we can unify these somewhat when we have the `Slab` as a model stub in the `src/` :)

Reply:
That's a good point!